it/uk/gov/hmrc/transitmovements/repositories/MovementsRepositorySpec.scala

```diff
@@ -235,11 +261,37 @@ class MovementsRepositorySpec
       )
     }
 
-  it should "return a list of movement responses for the supplied EORI if there are movements that were updated since the given time" in {
+  it should "return a list of an arrival movement responses for the supplied EORI if there are movements that were updated since the given time" in {
+    val dateTime = instant
+    GetMovementsSetup.setup()
+    await(repository.insert(GetMovementsSetup.arrivalGB3).value)
+    val result = await(repository.getMovements(GetMovementsSetup.eoriGB, MovementType.Arrival, Some(dateTime), None).value)
+
+    result.right.get.value should be(
+      NonEmptyList(
+        MovementWithoutMessages.fromMovement(GetMovementsSetup.arrivalGB2),
+        List(MovementWithoutMessages.fromMovement(GetMovementsSetup.arrivalGB1))
+      )
+    )
+  }
+
+  it should "return a list of an arrival movement responses for the supplied EORI if there are movements that matched with passed movementEORI" in {
```

Comment:
Can we set a different movement EORI for one of the movements so that we can properly test the filtering?

Reply:
Done